--- qiv-2.2.4/Makefile.orig	2012-02-22 10:53:37.202184976 +0100
+++ qiv-2.2.4/Makefile	2012-02-22 10:59:40.160218664 +0100
@@ -5,6 +5,8 @@
 
 # Directory where qiv will be installed under.
 PREFIX = /usr/local
+BINDIR = $(PREFIX)/bin
+MANDIR = $(PREFIX)/man
 
 # Font to use for statusbar in fullscreen mode
 STATUSBAR_FONT = "fixed"
@@ -47,7 +49,8 @@ MAGIC = -DHAVE_MAGIC
 
 CC        = gcc
 #CFLAGS    = -O -g -Wall
-CFLAGS    = -O2 -Wall \
+OPTS      = -O2
+CFLAGS    = $(OPTS) -Wall \
 	    -fcaller-saves -ffast-math -fno-strength-reduce \
 	    -fthread-jumps #-march=pentium #-DSTAT_MACROS_BROKEN
 #CFLAGS    = -O2 -Wall -fomit-frame-pointer -finline-functions \
@@ -83,7 +86,8 @@ OBJS_G   += lib/getopt.g lib/getopt1.g
 endif
 
 ifdef GTD_XINERAMA
-LIBS     += -L/usr/X11R6/lib -lXinerama
+INCLUDES += $(shell pkg-config --cflags xinerama)
+LIBS     += $(shell pkg-config --libs xinerama)
 endif
 
 ifdef MAGIC
@@ -127,21 +131,9 @@ distclean : clean
 
 install: $(PROGRAM)
 	@echo "Installing QIV..."
-	@if [ ! -e $(PREFIX)/bin ]; then \
-	  install -d -m 0755 $(PREFIX)/bin; \
-	  echo install -d -m 0755 $(PREFIX)/bin; \
-        fi
-	install -s -m 0755 $(PROGRAM) $(PREFIX)/bin
-	@if [ ! -e $(PREFIX)/man/man1 ]; then \
-	  echo install -d -m 0755 $(PREFIX)/man/man1; \
-	  install -d -m 0755 $(PREFIX)/man/man1; \
-	fi
-	install -m 0644 $(PROGRAM).1 $(PREFIX)/man/man1
-	$(COMPRESS_PROG) $(PREFIX)/man/man1/$(PROGRAM).1
-	@if ./qiv -f ./intro.jpg ; \
-	then echo "-- Test Passed --" ; \
-	else echo "-- Test Failed --" ; \
-	fi
+	install -d $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)/man1
+	install -s -m 0755 $(PROGRAM) $(DESTDIR)$(BINDIR)
+	install -m 0644 $(PROGRAM).1 $(DESTDIR)$(MANDIR)/man1
 	@echo "\nDont forget to look into the \"qiv-command\" file and install it!\n-> cp qiv-command.example /usr/local/bin/qiv-command\n\n"
 
 # the end... ;-)
